src/hotspot/share/opto/block.cpp

```diff
@@ -1356,6 +1356,9 @@ void PhaseCFG::verify() const {
       verify_memory_writer_placement(block, n);
       if (n->needs_anti_dependence_check()) {
         verify_anti_dependences(block, n);
+        if (C->failing()) {
```

Comment:
I don't see where `record_failure()` is called from `verify_anti_dependences()`.